app/views/layouts/application.html.slim

```diff
@@ -7,11 +7,16 @@ html
     = stylesheet_link_tag    "application", :media => "all"
     = csrf_meta_tags
     = javascript_include_tag "application"
-    meta name="viewport" content="width=device-width, initial-scale=1.0"
-    = render 'layouts/shim'
 
+    <script src="//maps.google.com/maps/api/js?v=3.23&key=AIzaSyCCe5VauNKbRn8PidIxl9wICa1RB2FKTP8"></script>
+
+    <script src="//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js"></script>
 
 
+    <script src="//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js"></script>
```

Comment:
Two references to the same file; also, is there another way to include this file? Rather than referencing rawgit.com, let's include this file in our app.